csharp.xUnit/GildedRose/GildedRose.cs

```diff
@@ -1,89 +1,89 @@
 ﻿using System.Collections.Generic;
 
-namespace GildedRoseKata
+namespace GildedRoseKata;
+
+public class GildedRose
 {
-    public class GildedRose
+    private readonly IList<Item> _items;
```

Comment:
Items are package access and mutable in other languages. Please do not change this. Also keep original name please

Reply:
This is already part of exercise